# Participant file used only in baseline.cpp
participant_file: "/home/ttsitos/catkin_ws/src/hrc_study_tsitosetala/games_info/baseline/EM"

# Limits of EE velocity
max_vel: 0.2
min_vel: -0.2

# Limits of EE position
max_x: -0.174 #-0.179
min_x: -0.356 #-0.359
max_y: 0.343
min_y: 0.162

# Starting positions
start_position_0: [-0.346, 0.333, 0.173] # down left
start_position_1: [-0.345, 0.172, 0.173] # down right
start_position_2: [-0.185, 0.332, 0.173] # up left 
start_position_3: [-0.184, 0.172, 0.173] # up right